src/Microsoft.HttpRepl/UserProfile/IUserProfileDirectoryProvider.cs

```diff
@@ -0,0 +1,7 @@
+namespace Microsoft.HttpRepl.UserProfile
+{
+    public interface IUserProfileDirectoryProvider
```

Comment:
Also not a fan of this name (or the implementing class). It IS descriptive at least...